pkg/env/notifier_secret_encryption.go

```diff
@@ -0,0 +1,9 @@
+package env
+
+var (
+	// EncryptNotifierSecrets controls if notifier secrets must be encrypted
+	EncryptNotifierSecrets = RegisterBooleanSetting("ROX_ENCRYPT_NOTIFIER_SECRETS", false)
```

Comment:
Once [this PR](https://github.com/stackrox/stackrox/pull/7897) is merged, you should use that